web/client/configs/localConfig.json

```diff
@@ -215,7 +215,8 @@
 						"group": "background",
 						"title": "Empty Background",
 						"fixed": true,
-						"type": "empty"
+						"type": "empty",
+						"visibility": false
```

Comment:
technically this is not needed to fix the issue, but it makes sense to have it aligned with the new.json